packages/fiori-freestyle-writer/src/index.ts

```diff
@@ -88,7 +88,8 @@ async function generate<T>(basePath: string, data: FreestyleApp<T>, fs?: Editor)
     // Add service to the project if provided
     if (ffApp.service) {
         await addOdataService(basePath, ffApp.service, fs);
-    } else {
+    } else if (!(ffApp.app?.projectType === 'CAPJava' || ffApp.app?.projectType === 'CAPNodejs')) {
```

Comment:
Can we please use the positive case:
```suggestion
    if (projectType ===  'EDMXBackend' ) {
```